browser/main/modals/PreferencesModal/HotkeyTab.js

```diff
@@ -70,6 +72,15 @@ class HotkeyTab extends React.Component {
     this.setState({
       config
     })
+    if (JSON.stringify(this.oldHotkey) === JSON.stringify(config.hotkey)) {
```

Comment:
Couldn't you write `this.oldHotkey === config.hotkey`?

Reply:
No, I can't, because object are passed by reference, then the "===" operator is checking if both variables got the same address, since not, it can't work.